### PR TITLE
Added debug logging of activity to the g8os

### PR DESCRIPTION
**Pull request by FastGeert**

### CODE REVIEW

File: pyclient/g8core/client.py [hunk 1]
```diff
@@ -5,11 +5,16 @@
...
 import signal
+import socket
+import logging
+import time
 from g8core import typchk
 
 
 DefaultTimeout = 10  # seconds
 
+g8logger = logging.getLogger('g8core')
```
Comment (zaibon) [inline pyclient/g8core/client.py]: convention use to name this simply `logger`

File: pyclient/g8core/client.py [hunk 2]
```diff
@@ -98,11 +103,18 @@ def get(self, timeout=None):
...
-        payload = json.loads(v.decode())
-        return Return(payload)
+        start = time.time()
+        maxwait = timeout
+        while maxwait > 0:
+            v = r.brpoplpush(self._queue, self._queue, 10)
+            if not v is None:
+                payload = json.loads(v.decode())
+                r = Return(payload)
+                g8logger.debug('%s << %s, stdout="%s", stderr="%s", data="%s"', self._id, r.state, r.stdout, r.stderr, r.data)
```
Comment (zaibon) [inline pyclient/g8core/client.py]: printing data can be dangerous here, you may print MB of data to the console.

Comment (FastGeert) [inline pyclient/g8core/client.py]: Ok, truncate it to the 1st 1000 characters?

File: pyclient/g8core/client.py [hunk 3]
```diff
@@ -98,11 +103,18 @@ def get(self, timeout=None):
...
-        v = r.brpoplpush(self._queue, self._queue, timeout)
-        if v is None:
-            raise Timeout()
-        payload = json.loads(v.decode())
-        return Return(payload)
+        start = time.time()
+        maxwait = timeout
+        while maxwait > 0:
+            v = r.brpoplpush(self._queue, self._queue, 10)
+            if not v is None:
```
Comment (muhamadazmy) [inline pyclient/g8core/client.py]: pep8 recommends `if v is not None` over `if not v is None`

**Pull request merged**